locallib.php

```diff
@@ -961,6 +954,14 @@ public function take_from_form_data($formdata) {
         $event->add_record_snapshot('attendance_sessions', $session);
         $event->trigger();
 
+    }
+
+    function redirect_after_take_from_form($formdata) {
```

Comment:
is $formdata used in this function? - if not we shouldn't include it as a param.